src/status_im/chat/commands/impl/map.cljs

```diff
@@ -0,0 +1,93 @@
+(ns status-im.chat.commands.impl.map
```

Comment:
Could this be a sub ns of the extension one?

Reply:
ok!

Reply:
what about make it a a common status component under `status-im.ui.components`?